docker/docker.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	DockerRegistryURL = "docker.io"
 	RestartTimeout    = 30
+	StopTimeout       = 30
```

Comment:
nit: although it's kinda obvious, lets put a comment that it's in seconds

Reply:
will do